experiment/find-merge-conflicts.sh

```diff
@@ -93,7 +93,8 @@ for pr in ${conflicting_prs[@]}; do
   git merge --ff FETCH_HEAD --strategy resolve >/dev/null 2>/dev/null || true
   echo "* ${ORG}/${REPO}/pull/${pr}:"
   # Report files that are conflicting
-  PAGER=cat git diff --name-only --diff-filter=UXB
+  export PAGER
```

Comment:
I typically set the var then export it, but that's a style nit

Reply:
shellcheck actually suggests this, FWIW https://github.com/koalaman/shellcheck/wiki/SC2155

Reply:
we have errexit enabled so it's sort of a style thing really as long as they are seperate statements, but I'd prefer define-then-export as well.